.github/PULL_REQUEST_TEMPLATE/modify-pep.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Update/modify PEP
+about: Add to or change an existing PEP
+---
+
+<!--
+**Please** read our Contributing Guidelines (CONTRIBUTING.rst)
+to make sure this repo is the right place for your proposed change. Thanks!
+-->
+
+* Change is either:
+    * [ ] To a Draft PEP
+    * [ ] To an Accepted or Final PEP, with Steering Council approval
+    * [ ] Resolving a purely technical or editorial issue
```

Comment:
I'd read that as resolving technical issues in the proposal. This should only be about issues in how the proposal is described :)
```suggestion
    * [ ] Fixing an editorial issue (e.g. typo)
```

Reply:
Yeah, I went back and forth on that, but yeah you're right—it's best to avoid misinterpretation, and generally speaking we're going to be the ones fixing technical issues of the sort I meant (warnings, links, Sphinx syntax, headers, etc.) which arguably fall under the general umbrella of editorial issues anyway.

I do suggest if we mention examples, we do so more broadly than jsut typos, or we may encourage users spending their time going looking for them, like we've had a number of PRs of in the past (perhaps even running automated tooling) as opposed to much more useful things like the stuff I list in https://github.com/python/peps/issues/10#issuecomment-1382420957 (and will list and describe in more detail in the forthcoming meta-issue mentioned there).

So, e.g.:

```suggestion
    * [ ] To fix an editorial issue (markup, typo, link, header, etc)
```

Reply:
Going ahead with this suggestion for now